src/sonic-host-service/sonic_host_server.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+"""Host Service to handle docker-to-host communication"""
+
+import os
+import os.path
+import glob
+import importlib
+import sys
+
+import dbus
+import dbus.service
+import dbus.mainloop.glib
+
+from gi.repository import GObject
+
+BUS_NAME = 'org.SONiC.HostService'
+BUS_PATH = '/org/SONiC/HostService'
+
+def register_modules():
```

Comment:
Can you point us a sample to use sonic-host-service ? for ex: from rest server/cli rebooting or running image upgrade ?

Reply:
This is one that we were using in the past. The host module registers the `org.SONiC.HostService.showtech.info` RPC, and it gets called from translib.

* [Host module](https://github.com/project-arlo/sonic-mgmt-framework/blob/0fdbb3dc420f609c326c2fc7b58ae47b2d63a427/scripts/host_modules/showtech.py)
* [Translib call to host services](https://github.com/project-arlo/sonic-mgmt-framework/blob/0fdbb3dc420f609c326c2fc7b58ae47b2d63a427/src/translib/transformer/xfmr_showtech.go#L68)

Reply:
By any chance is there any python client interface/ example to call host service?

Reply:
We don't have a python client at the moment, but it shouldn't be too hard. The host service uses D-Bus under the hood, so you only need to follow the python D-Bus examples to call the RPCs exposed by the individual applets.

Reply:
Ok, thanks.